sources/packages/backend/apps/api/src/services/notification/gc-notify.model.ts

```diff
@@ -52,3 +52,10 @@ export interface MinistryStudentFileUploadPersonalization {
   givenNames: string;
   lastName: string;
 }
+
+export interface StudentRestrictionAddedPersonalization {
```

Comment:
`added personalization`. i didn't get the context

Reply:
As we talked, the `Personalization` represents the variables in the GC Notify that will be sent for a template as per GC Notify documentation: https://documentation.notification.canada.ca/en/send.html#sending-an-email